vagrant_scripts/install_python.sh

```diff
@@ -0,0 +1,2 @@
+/home/vagrant/.pyenv/bin/pyenv install 3.9.5
+/home/vagrant/.pyenv/bin/pyenv global 3.9.5
```

Comment:
You shouldn't need the full path to pyenv as you add it to your PATH in .profile.